libs/admin/src/lib/components/users/users.component.ts

```diff
@@ -16,15 +20,33 @@ import { selectUsers, selectWorkgroups } from '../../state/admin.selector';
   styleUrls: ['./users.component.scss'],
   standalone: false,
 })
-export class UsersComponent implements OnInit, OnDestroy {
+export class UsersComponent implements OnInit, OnDestroy, AfterViewInit {
+  private users: User[] = [];
+
+  public showFilters = false;
   public workgroups = new Map<WorkgroupId, Workgroup>();
+  public workgroupFilterValues: Filter[] = [];
+  public readonly langFilterValues: Filter[] = [
+    { displayValue: 'DE', value: 'de' },
+    { displayValue: 'EN', value: 'en' },
+    { displayValue: 'FR', value: 'fr' },
+    { displayValue: 'IT', value: 'en' },
+  ];
+  public isAdminFilterValues: Filter[] = [];
 
-  protected readonly COLUMNS = ['email', 'isAdmin', 'languages', 'workgroups', 'actions'];
+  protected readonly COLUMNS = ['firstName', 'lastName', 'email', 'workgroups', 'isAdmin', 'languages', 'actions'];
   protected readonly WORKGROUP_DISPLAY_COUNT = 3;
 
+  protected dataSource: MatTableDataSource<User> = new MatTableDataSource<User>();
+  @ViewChild(MatPaginator) protected paginator!: MatPaginator;
+  private readonly searchTerm$ = new BehaviorSubject<string>('');
+  private readonly activeFilters$ = new BehaviorSubject<Map<keyof User, (string | number | boolean)[]>>(new Map());
```

Comment:
When using composite types, I prefer to use `Array<...>` over `...[]`, as it is very easy to not see that the latter is there.
```suggestion
  private readonly activeFilters$ = new BehaviorSubject<Map<keyof User, Array<string | number | boolean>>>(new Map());
```
In general, the longer a type name gets, the more I'm leaning towards using `Array<...>`.
There is even a [eslint rule](https://typescript-eslint.io/rules/array-type/#array-simple) for this, which some projects se to enforce this behavior.